configs/sklearn/trees.json

```diff
@@ -0,0 +1,104 @@
+{
+    "PARAMETER_SETS": {
+        "random forest implementations": {
+            "algorithm": [
+                { "library": "sklearn", "device": "cpu" },
+                { "library": "sklearnex", "device": "cpu" },
+                { "library": "sklearnex.preview", "device": ["cpu", "gpu"] }
+            ]
+        },
+        "extra trees implementations": {
+            "algorithm": [
+                { "library": "sklearn", "device": "cpu" },
+                { "library": "sklearnex.preview", "device": ["cpu", "gpu"] }
+            ]
+        },
+        "common ensemble params": {
+            "algorithm": {
+                "estimator_params": {
+                    "n_estimators": 200,
+                    "max_depth": null,
+                    "min_samples_split": 5,
+                    "max_features": "sqrt",
```

Comment:
This is a classification only parameter, regression should be "max_features": 1.0  . This heavily impacts performance, (should make things worse, but 1.0 is standard for RandomForestRegressor and ExtraTreesRegressor)

Reply:
Applied.